synth.py

```diff
@@ -13,9 +13,12 @@
 # limitations under the License.
 
 """This script is used to synthesize generated parts of this library."""
+# Copy of Resource name helper classes from v1 and import replacements
+# will be removed before next major release
```

Comment:
Consider marking the specific sections that can be removed after the change above, so that we'll know to keep lines 43 and 70+.